jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerClient.java

```diff
@@ -75,13 +109,23 @@ public boolean isDockerInstalled() {
    *     href="https://docs.docker.com/engine/reference/commandline/load/">https://docs.docker.com/engine/reference/commandline/load/</a>
    * @return stdout from {@code docker}
    */
-  public String load(Blob imageTarballBlob) throws IOException, InterruptedException {
+  public String load(Blob imageTarballBlob) throws InterruptedException, IOException {
     // Runs 'docker load'.
     Process dockerProcess = docker("load");
 
-    try (OutputStream stdin = dockerProcess.getOutputStream()) {
-      imageTarballBlob.writeTo(stdin);
-    }
+    writeToStdin(
```

Comment:
This new method feels like it adds a high cost to readability/code complexity without enough benefit. It feels like we're decoupling the exception handling and method calls in an unnatural way. I would prefer if we kept this simple here (was this for testing)

Reply:
Okay, I moved everything back to `load`.